std/concurrency.d

```diff
@@ -106,6 +106,30 @@ private
         static assert(!hasLocalAliasing!(SysTime, Container));
     }
 
+    bool hasLocalReferencePassing(F, Types...)()
```

Comment:
Probably this should be made private. Otherwise it needs to be ddoc'ed.

Reply:
Yes, it's under this private block https://github.com/dlang/phobos/pull/7124/files#diff-25bf2af9a45af72e9499b850a56d6f32R73